### PR TITLE
Added details

### PR DESCRIPTION
**Pull request by roshan1337d**
Rudradev Pradhan

### CODE REVIEW

File: b121047 [hunk 1]
```diff
@@ -0,0 +1 @@
+Rudradev Pradhan
```
Comment (clashverse) [inline b121047:1]: Remove the space between firstname and lastname

**Review by clashverse — CHANGES_REQUESTED** [review]
Modify the name

File: b121047 (hunk 1 above)
Comment (clashverse) [reply b121047:1]: Ok, doing it soon!

Comment (roshan1337d) [reply b121047:1]: Ok doing it soon!

**Review by clashverse — APPROVED** [review]
All right, can be merged now!

**Pull request merged**